### PR TITLE
fix(Core/Boss): Boss Urom

### PR DESCRIPTION
**Pull request by poszer**
- Should fix issue with not spawned

Boss Urom teleporting issue

##### CHANGES PROPOSED:

-  Updated
-  

###### ISSUES ADDRESSED:

Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1957

##### TESTS PERFORMED:

Yes, builded without errors. Tested in game 15 Runs, for me it works fine. In all 15 runs , the Boss is correctly teleported, spawned.

##### HOW TO TEST THE CHANGES:

1. Do 5H dungeon several times

##### KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

##### Target branch(es):

Master

### CODE REVIEW

File: src/server/scripts/Northrend/Nexus/Oculus/boss_urom.cpp [hunk 1]
```diff
@@ -265,8 +266,11 @@ class boss_urom : public CreatureScript
                     
                     Talk(SAY_ARCANE_EXPLOSION);
                     Talk(EMOTE_ARCANE_EXPLOSION);
-
-                    me->CastSpell(me, SPELL_EMPOWERED_ARCANE_EXPLOSION, false);
+					
+                    if (me->GetMap()->IsHeroic())
+						DoCastAOE(SPELL_EMPOWERED_ARCANE_EXPLOSION_H);
+					else
+					me->CastSpell(me, SPELL_EMPOWERED_ARCANE_EXPLOSION, false);
```
[the remark this answers is not in the document]
Comment (poszer) [reply src/server/scripts/Northrend/Nexus/Oculus/boss_urom.cpp]: Done

**Comment by spinzaku** [comment]
Great work! testing tonight

**Comment by poszer** [comment]
> Confirmed the boss now spawns properly

Nice, thank you for testings @heyaapl

**Comment by Knindzagxg** [comment]
Tested in-game, everything working well.

**Comment by spinzaku** [comment]
tested worked!!

**Comment by poszer** [comment]
Thanks for testings guys, this is ready

**Pull request merged**